Added new backuping option.

thecsw opened the pull request:
This little script will dump the whole database into a backup file with a timestamp. The location of the file is volumed with the main machine.

thecsw committed:
Added new backuping option.

This little script will dump the whole database into a backup file with a timestamp. The location of the file is volumed with the main machine.

rickles42 reviewed (changes requested):
Your approach uses `docker-compose run` to launch a new temporary database container and call `mysqldump` on it. I think it might be safer to instead use `docker-compose exec` to just execute `mysqldump` on the real database container. I feel like having two containers accessing the /data/memeinvestor/mysql volume at the same time could lead to problems or at least confusion.

Another question, would it be possible to use `docker copy` to copy the SQL dump file out of the container onto the host, instead of mapping a new volume? Mapping that backups folder means that all the backups are a bit more exposed to the container instead of safe and sound tucked away in a folder on the host. I don't think it's really much of a risk but it seems like keeping the backups out of the hands of the running system is a better approach.

thecsw commented:
Sure it is! On it.

thecsw committed:
Made a few adjustments.

Instead of voluming backups, we take them and copy from docker containers to the host machine.

thecsw commented:
Take a look

thecsw committed:
Added newline in override file.

thecsw committed:
I am sorry about this. Ended file with a newline.
